src/components/biddingPage/graph/lineChart.tsx

```diff
@@ -98,28 +98,32 @@ const LineChart: React.FC<IProps> = ({ dataBuy, dataSell }) => {
       now.getMonth() + 1
     ).toString()}/${now.getDate().toString()}`;
     dataBuy.map((d) => {
-      if (d.date === dateStr && d.time === validTime && validTime !== 0) {
+      if (
+        new Date(d.date).getTime() === new Date(dateStr).getTime() &&
```

Comment:
I think you can simply use `dayjs()` to parse variables.

Reply:
Since `dayjs` can't parse a `YYYY/MM/DD` formatted date `d` by using `dayjs(d)`, I'll need to parse `d` into `YYYY-MM-DD` format first to apply `dayjs`. Hence, I think this is the simplest way because both `d.date` and `dateStr` are in `YYYY/MM/DD` format.

Reply:
code
![image](https://user-images.githubusercontent.com/58351529/101053476-caf65180-35c2-11eb-806a-77193360b810.png)
in chrome
![image](https://user-images.githubusercontent.com/58351529/101053562-e3ff0280-35c2-11eb-9e37-44ad911b7d36.png)

Reply:
Ok, I got it.
Thx.